### PR TITLE
agent return code

### PR DESCRIPTION
**Pull request by TingluoHuang**

### CODE REVIEW

File: src/Microsoft.VisualStudio.Services.Agent/Constants.cs [hunk 1]
```diff
@@ -26,6 +26,14 @@ public static class Agent
             public static readonly OSPlatform Platform = OSPlatform.Windows;
 #endif
 
+            public static class ReturnCode
+            {
+                public const int Zero = 0;
```
Comment (bryanmacfarlane) [inline src/Microsoft.VisualStudio.Services.Agent/Constants.cs]: Shouldn't this be ReturnCode.Success?

File: src/Misc/layoutbin/en-US/strings.json [hunk 2]
```diff
@@ -163,6 +163,7 @@
   "Password": "password",
   "PathNotExist": "Path not exist: {0}",
   "PersonalAccessToken": "personal access token",
+  "PlatformNotSupport": "Current platform is not capable of running agent that build for {0}.",
```
Comment (bryanmacfarlane) [inline src/Misc/layoutbin/en-US/strings.json]: Running the agent on this platform is not supported.  The current platform is {0} and it was built for {1}

File: src/Agent.Worker/Build/GitCommandManager.cs [hunk 3]
```diff
@@ -72,6 +72,13 @@ public class GitCommandManager : AgentService, IGitCommandManager
             }
         };
 
+        private readonly Dictionary<string, string> _userAgentEnv = new Dictionary<string, string>()
+        {
+            {
+                "GIT_HTTP_USER_AGENT", "vstsagentcore_git"
```
Comment (bryanmacfarlane) [inline src/Agent.Worker/Build/GitCommandManager.cs]: vsts-agent-git

**Comment by bryanmacfarlane** [comment]
Other than comments, looks good.

**Comment by TingluoHuang** [comment]
@bryanmacfarlane this is not the final PR, i pushed this for self-update testing. :D

File: src/Microsoft.VisualStudio.Services.Agent/Constants.cs (hunk 1 above)
Comment (TingluoHuang) [inline src/Microsoft.VisualStudio.Services.Agent/Constants.cs]: I like Success, i can't find a better name :D

**Comment by bryanmacfarlane** [comment]
Just confirming - we need this for the next preview, right?

**Comment by TingluoHuang** [comment]
@bryanmacfarlane Yes, better to have for preview.

**Pull request merged**